example_workspace/src/hello_world_example_app/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(hello_ros)
+
+find_package(catkin REQUIRED rosjava_build_tools)
```

Comment:
I think we need to add `rosjava_build_tools` to `ros.rosinstall`.

Reply:
Forgot that, it was already installed in my output dir.